statsmodels/iolib/summary2.py

```diff
@@ -395,7 +395,13 @@ def _col_params(result, float_format='%.4f', stars=True):
         res.loc[idx, res.columns[0]] = res.loc[idx, res.columns[0]] + '*'
     # Stack Coefs and Std.Errors
     res = res.iloc[:, :2]
-    res = res.stack()
+    if not isinstance(result, MixedLMResultsWrapper):
```

Comment:
Why don't you just duck-type:

``` 
if hasattr(result,'rsquared'):
    # Do stuff here with rsquared
if hasattr(result,'rsquared)adj'):
    # Do stuff here with rsquared-adj
```

This is much cleaner than making ex exemption to a single model, and having to create a dependency in this module to an unrelated one.